.github/ISSUE_TEMPLATE/experimental.md

```diff
@@ -0,0 +1,32 @@
+name: Create a Bug (English)
+description: Report an issue in English
+body:
+  - type: markdown
+    id: problem
+    attributes:
+      label: What Happend?
+      description: |
+      Tip: Add the "--alsologtostderr" flag to the command-line for more logs
+    validations:
+      required: true
+  - type: markdown
+    id: problem
+    attributes:
+      description: |
+      label: Attach log file
```

Comment:
I believe you want this label above `description` because I'm assuming the `Tip` line above is what you want in the description.

Reply:
u are right